plugins/catalog-backend/examples/locationref.yaml

```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: backstage.io/v1beta1
+kind: LocationRef
+metadata:
+  name: location-1
```

Comment:
we still do not have a good mechanism for automatic naming of entities - for now, it will be up to the user to name these uniquely

Reply:
How would that automatic naming even work though? 😅  What if you list 3 locations and let the names be generated, we then need a bunch of logic to try to figure out which one was removed? Assuming we want to keep generation correct and so on.

Would probably rename this one to `location-example-components`?

Reply:
Yeah so, the automatic naming probably won't be a good fit for file based things, more for entities that you POST directly to the service. Which the frontend totally may do, I mean, in the register flow if you point to a component file, the frontend could either POST the component file directly (and it would get an auto generated annotation of where it came from), oooor it could generate a Location entity in-memory that points to the component file, and POST that - in such a way that the name of the Location entity is auto generated because effectively it doesn't matter.

Same thing for my monorepo root location entity idea - the user would paste an URL to the root of the monorepo, and the Backstage frontend would just generate a RootLocation or whatever entity and POST that.

Reply:
And yeah that doesn't apply to the use case that this particular user wanted, with file based location yamls too.